scripts/spark-tests/plugins/spark.py

```diff
@@ -201,6 +201,10 @@ class TestMarker:
         keywords=["SparkSessionTestCase", "test_session.py"],
         reason="Client keeps retrying setting session config for invalid remote endpoints",
     ),
+    TestMarker(
+        keywords=["JobCancellationTests", "test_session.py"],
+        reason="Slow test not fully working yet",
+    ),
```

Comment:
This slow test now becomes noticeable since it uses `.mapInPandas()`, which this PR supports.